### PR TITLE
A trial career test

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Just making sure this looks about right for how we want to test the Career pages

## Additions

- Added tests for careers/students-and-graduates

## Changes

- Changed some of the the-bureau tests to make them pass

## Testing

- Run a protractor test.

## Review

- @jimmynotjim 
- @anselmbradford 
- @sebworks

### CODE REVIEW

File: _tests/browser_tests/page_objects/page_careers-students-and-graduates.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+function careers_studentsAndGraduates() {
```
Comment (anselmbradford) [inline _tests/browser_tests/page_objects/page_careers-students-and-graduates.js]: The underscore in the function name will give a linter error. If we want to make underscores a convention in the test JS there just needs to be an eslintrc override file when https://github.com/cfpb/cfgov-refresh/issues/723 is closed.

Comment (KimberlyMunoz) [inline _tests/browser_tests/page_objects/page_careers-students-and-graduates.js]: I'm happy to change the name to something else. Let's chat about how we want to handle the naming for subpages.

File: _tests/browser_tests/spec_suites/shared/shared_the-bureau.js [hunk 2]
```diff
@@ -16,4 +16,4 @@ describe( 'Beta The Bureau Page', function() {
     expect( page.missions.count() ).toEqual( 3 );
     expect( page.missions.getText() ).toEqual[ 'Educate', 'Enforce', 'Empower' ];
   } );
-} );
+} );
```
Comment (anselmbradford) [inline _tests/browser_tests/spec_suites/shared/shared_the-bureau.js]: Minor: keep EOF newline.

File: _tests/browser_tests/page_objects/page_careers-students-and-graduates.js (hunk 1 above)
Comment (anselmbradford) [inline _tests/browser_tests/page_objects/page_careers-students-and-graduates.js]: From convo, suggest to remove `careers_`

File: _tests/browser_tests/spec_suites/shared/shared_careers-students-and-graduates.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+var Careers_StudentsAndGraduates = require( '../../page_objects/page_careers-students-and-graduates.js' );
```
Comment (anselmbradford) [inline _tests/browser_tests/spec_suites/shared/shared_careers-students-and-graduates.js]: From convo, suggest to remove `Careers_`

File: _tests/browser_tests/spec_suites/shared/shared_the-bureau.js (hunk 2 above)
Comment (jimmynotjim) [inline _tests/browser_tests/spec_suites/shared/shared_the-bureau.js]: Finally added `"ensure_newline_at_eof_on_save": true` to my ST3 user settings

File: _tests/browser_tests/spec_suites/shared/shared_careers-students-and-graduates.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+  it( 'should properly load in a browser', function() {
+    expect( page.pageTitle() ).toBe( 'Students and Graduates' );
+  } );
+
+  it( 'should have 4 opportunities for students and graduates', function() {
+    expect( page.opportunities.count() ).toEqual( 4 );
+    expect( page.opportunities.getText() ).toContain( 'Presidential Management Fellow' );
+  } );
+
```
Comment (jimmynotjim) [inline _tests/browser_tests/spec_suites/shared/shared_careers-students-and-graduates.js]: Extra line break

**Comment by jimmynotjim** [comment]
YAY MOAR TESTS!!!

Minor comment, but :+1: after making @anselmbradford suggested updates

File: _tests/browser_tests/spec_suites/shared/shared_the-bureau.js (hunk 2 above)
Comment (anselmbradford) [inline _tests/browser_tests/spec_suites/shared/shared_the-bureau.js]: I just added this too. For @sebworks + @KimberlyMunoz :

- Sublime Text > Preferences > Settings -- User
Should look something like:
```
{
	"font_size": 17,
	"ignored_packages":
	[
		"Vintage"
	],
	"open_files_in_new_window": false,
	"ensure_newline_at_eof_on_save": true
}
```

Comment (jimmynotjim) [inline _tests/browser_tests/spec_suites/shared/shared_the-bureau.js]: I just shared my user settings in the GHE D&D Team repo, everyone should check it out and add their own.

**Comment by KimberlyMunoz** [comment]
I'm going to reopen this later. There's a lot of ESLint things I want to fix as well here.

**Pull request closed without merging**